src/scales/scale.category.js

```diff
@@ -60,7 +60,7 @@ module.exports = Scale.extend({
 			return me.getRightValue(chart.data.datasets[datasetIndex].data[index]);
 		}
 
-		return me.ticks[index - me.minIndex];
+		return me._getLabels()[index];
```

Comment:
Is altering of ticks in `ticks.callback` or `afterBuildTicks` supported?
`me.getTicks()[index].label` instead, in that case?

Reply:
Hmm, it's a good question. My view is that it wouldn't be supported because there's no other scale that accesses ticks in `getLabelForIndex` and this method is only used by the tooltip which a separate concept from ticks. The way we've always supported customizing the tooltip is via https://www.chartjs.org/docs/latest/configuration/tooltip.html?q=#external-custom-tooltips